developer-extension/src/panel/hooks/useEvents.ts

```diff
@@ -0,0 +1,89 @@
+import { useEffect, useState } from 'react'
+import { generateUUID } from '../../../../packages/core/src/tools/utils'
+import type { StoredEvent } from '../../common/types'
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏remove the `StoredEvent` from `common`, since it is not used in the background anymore

Reply:
Sure